features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -211,8 +212,16 @@ internal class RumSessionScope(
         sessionState = if (keepSession) State.TRACKED else State.NOT_TRACKED
         sessionId = UUID.randomUUID().toString()
         sessionStartNs.set(nanoTime)
+        val backgroundEventTracking = sdkCore.getFeature(Feature.RUM_FEATURE_NAME)
+            ?.unwrap<RumFeature>()
+            ?.configuration?.backgroundEventTracking ?: false
+        sessionEndedMetricDispatcher.startMetric(
+            sessionId = sessionId,
+            startReason = reason,
+            ntpOffsetAtStart = sdkCore.time.serverTimeOffsetMs,
+            backgroundEventTracking = backgroundEventTracking
+        )
```

Comment:
blocking: I don't think you need `backgroundEventTracking` variable here, because there is `RumSessionScope#backgroundTrackingEnabled` property already.

```suggestion
        sessionEndedMetricDispatcher.startMetric(
            sessionId = sessionId,
            startReason = reason,
            ntpOffsetAtStart = sdkCore.time.serverTimeOffsetMs,
            backgroundEventTracking = backgroundTrackingEnabled
        )
```

Reply:
Indeed, fixed